src/Components/Scenes/SideNavigation/RequestListItem/RequestListItem-Container.js

```diff
@@ -3,7 +3,11 @@ import RequestListItem from './RequestListItem';
 
 const mapStateToProps = () => {
   return {
-    requestListItemsName: ['Inbox', 'Sent', 'Archive']
+    requestListItemsName: [
+      { label: 'inbox', link: '/requests' },
```

Comment:
Pulled down your branch and ran it locally.  FYI the labels for these are now lowercase instead of uppercase in `dev`

Reply:
Great catch I must have lowercased when I re-typed by acdient.